api/src/main/java/com/alibaba/nacos/api/ai/model/mcp/McpServerImportRequest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 1999-2021 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacos.api.ai.model.mcp;
+
+import java.io.Serializable;
+
+/**
+ * MCP Server Import Request.
+ *
+ * @author nacos
+ */
+public class McpServerImportRequest implements Serializable {
```

Comment:
Add Unit test for api request.

you can ref other tests in `api/src/main/java/com/alibaba/nacos/api/ai/model/mcp`